src/account/sync/mod.rs

```diff
@@ -544,7 +538,14 @@ async fn perform_sync(
                 )
                 .await?
             } else {
-                sync_addresses(&account, address_index, gap_limit, options).await?
+                let (found_public_addresses, mut messages) =
```

Comment:
Why don't you also make found_public_addresses mutable here?

Reply:
just for clarity on the variable name